py3status/modules/xrandr.py

```diff
@@ -248,15 +250,16 @@ def _apply(self, force=False):
             #
             if output in combination:
                 pos = getattr(self, '{}_pos'.format(output), '0x0')
+                rotation = getattr(self, '{}_rotate'.format(output), 'normal')
```

Comment:
it is probably worth checking that the value or rotation is legal eg in  ['normal', 'inverted', 'left', 'right']